index.html

```diff
@@ -8,6 +8,96 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+   <div id="home-page"> 
+        <div id="top-info">
+            <div id="top-left-info">
+                <div class="left-info">Travel</div>
+                <div class="left-info">Goo</div>
+            </div>
+            <div id="top-mid-info">
+               <div class="mid-info">Home</div>
+               <div class="mid-info">About Us</div>
+               <div class="mid-info">Packages</div>
+            </div>
+            <div id="top-right-info">
+                <img class="right-info" src="./images/star.svg">
+                <img id="circle" src="./images/profile.svg">
+            </div> 
+        </div> 
+        <div id="centralised-info">
+            <div id="mid-quote">
+                <div class="quote">Your Imagination Is</div>
+                <div id="line1" class="quote">Your Only limit</div>
+            </div>
+            <div id="our-role">
+                <div>We always try to make our customer Happy, We provide all kind of facilities</div>
+                <div id="line2">Your Satisfaction is our main priority</div>
+            </div>
+            <div id="button">
+                <div id="green-box">Discover more</div>
```

Comment:
this should be an HTML button element. not div.

Reply:
yes akhi changed

Reply:
then please click on 'resolve conversation' and close the issue